Dockerfile

```diff
@@ -111,6 +111,10 @@ COPY . .
 # Install Python requirements again in order to capture local projects
 RUN pip install -e .
 
+RUN useradd -m --shell /bin/false app
+
+USER app
```

Comment:
Could we have a better name for this? like `celery_app` or `worker_user`?

Reply:
@schenedx it can simply be app. Because the same container image will be used to run either a gunicorn or celery, not both at the same time in the same container.